Makefile

```diff
@@ -534,7 +534,13 @@ cpplint:
 	--exclude_path src/operator/contrib/ctc_include
 
 pylint:
-	pylint --rcfile=$(ROOTDIR)/ci/other/pylintrc --ignore-patterns=".*\.so$$,.*\.dll$$,.*\.dylib$$" python/mxnet tools/caffe_converter/*.py
+	python3 -m pip install --upgrade --user pylint  # v1.8.3 --> v2.1.1
```

Comment:
We pin the versions on purpose and don't want them to be upgraded like that. Otherwise, an update of pylint may result in new errors being thrown and all pipelines are suddenly blocked.

https://github.com/apache/incubator-mxnet/blob/master/ci/docker/install/ubuntu_python2.sh
https://github.com/apache/incubator-mxnet/blob/master/ci/docker/install/ubuntu_python3.sh